dbytecoderc/Questioner
# #163421750 Delete meetup

#### What does this PR do?
> This PR give an admin the ability to delete a meetup 

#### Description of Task to be completed?
>Have DELETE (`/api/v1/meetups/meetupid`) route working

#### How should this be manually tested?
> - Clone the repo and run `npm install` to install dependencies.
> - Download and install postgresql from this [link](https://www.postgresql.org/)
> - Setup postgreql using this [guide](https://www.postgresql.org/docs/10/runtime.html)
> - run `npm migrate` to create the tables
> - run `npm start:dev` to start server
> - check the `server/test/testData` folder route for the input formats and proceed to testing with postman
> - alternatively run `npm run test` to test the endpoint

#### What are the relevant pivotal tracker stories?
> - ` #163421750`
